darwin/cli_functions.py

```diff
@@ -850,25 +851,30 @@ def delete_files(dataset_slug: str, files: List[str], skip_user_confirmation: bo
 
     Parameters
     ----------
-    dataset_slug: str
+    dataset_slug : str
         The dataset's slug.
-    files: List[str]
+    files : List[str]
         The list of filenames to delete.
-    skip_user_confirmation: bool
-        If True, skips user confirmation, if False it will prompt the user. Defaults to False.
+    skip_user_confirmation : bool, default: False
+        If ``True``, skips user confirmation, if False it will prompt the user.
     """
     client: Client = _load_client(dataset_identifier=dataset_slug)
     try:
-        console = Console()
+        console = Console(theme=_console_theme(), stderr=True)
         dataset: RemoteDataset = client.get_remote_dataset(dataset_identifier=dataset_slug)
-        items: Iterator[DatasetItem] = dataset.fetch_remote_files({"filenames": ",".join(files)})
+        items, items_2 = tee(dataset.fetch_remote_files({"filenames": ",".join(files)}))
```

Comment:
I thought we added support for passing list in directly (to prevent issues like `filename,with,commas.png`
Can you test:
```suggestion
        items, items_2 = tee(dataset.fetch_remote_files({"filenames": files}))
```

Reply:
Changed and tested. Seems to work indeed.